matlab/src/matlab/+arrow/recordBatch.m

```diff
@@ -16,7 +16,7 @@
 % permissions and limitations under the License.
 function rb = recordBatch(T)
```

Comment:
Since you renamed the first input argument to `matlabTable` in the `arguments` block, you will also need to rename `T` to `matlabTable`.

Reply:
Also, the rest of the code in this function needs to be updated to use `matlabTable` instead of `T`. For example:


```matlab
columnNames = string(T.Properties.VariableNames);
```

would become:

```matlab
columnNames = string(matlabTable.Properties.VariableNames);
```